.pre-commit-config.yaml

```diff
@@ -22,6 +22,7 @@ repos:
         exclude_types: [svg]
       - id: check-yaml
       - id: check-added-large-files
+        exclude: &exclude_pattern 'iv_weak_instruments.ipynb'
```

Comment:
Am I seeing right that the use of `rasterize` hasn't actually impacted the filesize of the notebook?

Reply:
I think it dropped it from 6.5 to 3.5 but not enough to avoid the large file check so i had this...

Reply:
oh, well I'll certainly take that drop in filesize 👍🏻